website/project/model.py

```diff
@@ -2359,15 +2371,19 @@ def add_log(self, action, params, auth, foreign_user=None, log_date=None, save=T
             user=user,
             foreign_user=foreign_user,
             params=params,
+            node=self,
+            original_node=params['node']
         )
 
         if log_date:
             log.date = log_date
+        log.save()
 
-        self.date_modified = log.date.replace(tzinfo=None)
+        if len(self.logs) == 1:
```

Comment:
Sorry for the novel :) changed to `self.date_modified = log.date.replace(tzinfo=None)`